### PR TITLE
Create Word completion given part of word.md

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

File: tasks/Word completion given part of word.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Summary
+Groups were given a set of 36 words with 2-3 letters missing (ex." _ ech_ _ que" would be "technique"). Groups had 10 minutes to complete as many words as possible. Groups earned 1 point for each correctly spelled English word they completed. 
+
+
+# References 
+Main Paper: https://www.science.org/doi/abs/10.1126/science.1193147
```
Comment (benjamin-manning) [inline tasks/Word completion given part of word.md:6]:
```suggestion
Main Paper: https://www.science.org/doi/abs/10.1126/science.1193147

Task Source: McGrath, Joseph Edward. Groups: Interaction and performance. Vol. 14. Englewood Cliffs, NJ: Prentice-Hall, 1984. http://users.ece.utexas.edu/~perry/education/382v-s08/papers/mcgrath84.pdf
```

File: tasks/Word completion given part of word.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+reached out to instructors 
+
+
+# Procedure
+## Steps
+1. When participants arrived at the laboratory they were randomly assigned to participate in a team or, in the event of overscheduling, randomly selected for exclusion. 
+2. Before meeting their teams, individuals completed a brief intelligence test (described below). 
+3. Following the intelligence test, teams began working together on the group tasks in a private laboratory room.
+4. Groups had ten minutes to complete as many of the 36 words they could 
+
```
Comment (benjamin-manning) [inline tasks/Word completion given part of word.md:19]:
```suggestion
4b. Other versions of the task use a different time period and different numbers of words, but the other features are the same. Additionally, this task comes from McGrath's broader set of tasks (quadrant 2 in his work) that are intellective: solving problems with a correct answer.
```

File: tasks/Word completion given part of word.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Summary
+Groups were given a set of 36 words with 2-3 letters missing (ex." _ ech_ _ que" would be "technique"). Groups had 10 minutes to complete as many words as possible. Groups earned 1 point for each correctly spelled English word they completed. 
+
+
+# References 
+Main Paper: https://www.science.org/doi/abs/10.1126/science.1193147
+Supplemental Materials: https://www.science.org/doi/suppl/10.1126/science.1193147/suppl_file/woolley_som.pdf
+
```
Comment (benjamin-manning) [inline tasks/Word completion given part of word.md:8]:
```suggestion
Alternative task method: http://www.craiganderson.org/wp-content/uploads/caa/Scales/WordComp.pdf
```

File: tasks/Word completion given part of word.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+# Procedure
+## Steps
+1. When participants arrived at the laboratory they were randomly assigned to participate in a team or, in the event of overscheduling, randomly selected for exclusion. 
+2. Before meeting their teams, individuals completed a brief intelligence test (described below). 
+3. Following the intelligence test, teams began working together on the group tasks in a private laboratory room.
+4. Groups had ten minutes to complete as many of the 36 words they could 
+
+
```
Comment (benjamin-manning) [inline tasks/Word completion given part of word.md:20]:
```suggestion
# Alternative Procedure
## Steps

1. Participants arrive and are told participants that they were looking at a list of 98 words with letters missing (varying in number), and that their task was to fill in the blanks to make complete words by themselves
2. Participants were given three minutes to complete as much of the word task as
they could. 
```

File: tasks/Word completion given part of word.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+Instructions developed by CSS lab 
+
+You will be given a set of 36 words with 2-3 letters missing (ex." _ ech_ _ que" would be "technique"). You will have 10 minutes to complete as many
+words as possible.  Your group will earn 1 point for each correctly spelled English word you complete. 
+
+
+# Criteria
+## Performance calculation
+Groups recieved one point for every word they completely correctly  
+
```
Comment (benjamin-manning) [inline tasks/Word completion given part of word.md:33]:
```suggestion
Alternative:  Participant responses were coded into the following categories: aggressive words, neutral words, ambiguous words, and non-words.
```

**Review by benjamin-manning — CHANGES_REQUESTED** [review]

**Review by sarikasu — COMMENTED** [review]
Stimuli not available from researchers I added some words!

File: tasks/Word completion given part of word.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+# References 
+Main Paper: https://www.science.org/doi/abs/10.1126/science.1193147
+
+Task Source: McGrath, Joseph Edward. Groups: Interaction and performance. Vol. 14. Englewood Cliffs, NJ: Prentice-Hall, 1984. http://users.ece.utexas.edu/~perry/education/382v-s08/papers/mcgrath84.pdf
+Supplemental Materials: https://www.science.org/doi/suppl/10.1126/science.1193147/suppl_file/woolley_som.pdf
+Alternative task method: http://www.craiganderson.org/wp-content/uploads/caa/Scales/WordComp.pdf
+# Stimuli
+Stimuli is unavailable from researchers, stimuli developed by CSS lab 
+
+ocean: _ _cean
```
Comment (benjamin-manning) [inline tasks/Word completion given part of word.md:14]:
```suggestion
ocean:  _ _can            
```

File: tasks/Word completion given part of word.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+ocean: _ _cean            
+lackadaisical: lack_ _ _isical 
+disgusted: _ _ _gusted 
+control: con_ _ _l 
+color: _ _ lor 
+nonchalant : non_ _ _lant
+punishment: _ _nishment 
+analyze: ana_ _ze 
+downtown: do_ _ _wn
```
Comment (benjamin-manning) [inline tasks/Word completion given part of word.md:22]:
```suggestion
downtown: dow_ _ _wn
```

**Review by benjamin-manning — APPROVED** [review]

File: tasks/Word completion given part of word.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+eatable: _ _table 
+horrible: hor_ _ _le 
+reaction: _ _action 
+quirky: _ _irky 
+punishment: pun_ _ _ment 
+garrulous: _ _rulous
```
Comment (benjamin-manning) [inline tasks/Word completion given part of word.md:28]:
```suggestion
garrulous: _ _ _rulous 
```

**Review by benjamin-manning — APPROVED** [review]

**Pull request merged**